src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/SingleApprenticeshipApproved.cshtml

```diff
@@ -48,7 +48,7 @@
         </tr>
     </table>
     <p>
-        <a href="@Url.EmployerCommitmentsAction($"{Model.ViewApprovedApprenticeDetailsLink}")" draggable="false" class="govuk-link">
+        <a href="@Url.EmployerCommitmentsV2Action($"{Model.ViewApprovedApprenticeDetailsLink}")" draggable="false" class="govuk-link">
```

Comment:
no need for the `$"` string format shorthand here. Just pass `Model.ViewApprovedApprenticeDetailsLink`.